packages/react-log-viewer/src/LogViewer/LogViewer.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect, memo } from 'react';
+import { LogViewerContext } from './LogViewerContext';
+import { css } from '@patternfly/react-styles';
+import { LoggerRow } from './LogViewerRow';
+import { LogViewerToolbar } from './LogViewerToolbar';
+import { LOGGER_HEIGHT, DEFAULT_FOCUS, DEFAULT_SEARCH_INDEX } from './utils/constants';
+import { searchForKeyword, parseConsoleOutput } from './utils/utils';
+import { VariableSizeList as List, areEqual } from 'react-window';
+import styles from '@patternfly/react-styles/css/components/LogViewer/base';
+import '@patternfly/react-core/dist/styles/base.css';
+
+interface LoggerProps {
+  /* String data being sent by the consumer*/
```

Comment:
Comments are not formatted properly here.  Should be: 
`/** prop description */`